src/dags/basiskaart.py

```diff
@@ -36,12 +36,12 @@
 from sqlalchemy import create_engine
 from sqlalchemy.exc import SQLAlchemyError
 
-dag_id = "basiskaart"
-owner = "dataservices"
-variables = Variable.get(dag_id, deserialize_json=True)
-tables_to_create = variables["tables_to_create"]
-source_connection = "AIRFLOW_CONN_POSTGRES_BASISKAART"
-import_step = 10000
+DAG_ID: Final = "basiskaart"
+OWNER: Final = "dataservices"
+VARIABLES: Final = Variable.get(DAG_ID, deserialize_json=True)
```

Comment:
The `Variable.get` is doing a db-query. So, `VARIABLES` is not a real constant that is fully defined in code. But, well, what is the definition of a constant?

Reply:
A constant is a thing that always has the same value. That fits the bill :)